sections/shared/Layout/AppLayout/Header/UserMenu/UserMenu.tsx

```diff
@@ -56,6 +59,27 @@ const UserMenu: FC<UserMenuProps> = ({ isTextButton }) => {
 		[redeemableDeprecatedSynths?.totalUSDBalance]
 	);
 
+	/**
+	 * @dev We require this new provider since we need one connected to Ethereum
+	 * mainnet and NOT to the Optimism network.
+	 */
+	const infuraUrl = 'https://mainnet.infura.io/v3/';
+
+	let ensProvider = new ethers.providers.JsonRpcProvider(
+		infuraUrl + process.env.NEXT_PUBLIC_INFURA_PROJECT_ID
+	);
```

Comment:
Can you try
```
let ensProvider = new ethers.providers.InfuraProvider(
	null, process.env.NEXT_PUBLIC_INFURA_PROJECT_ID
);
```
and if that doesn't work
```
let ensProvider = new ethers.providers.InfuraProvider()
```
https://docs.ethers.io/v5/api/providers/api-providers/#InfuraProvider

Reply:
ill try that rn 🙂

Reply:
> > @JChiaramonte7 still seeing the same error message with the new `ensProvider`:
> > ```js
> > const apiKey = process.env.NEXT_PUBLIC_INFURA_PROJECT_ID;
> > 
> > let ensProvider = new ethers.providers.InfuraProvider(1, apiKey);
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > <img alt="Screen Shot 2022-03-02 at 1 20 43 PM" width="1024" src="https://user-images.githubusercontent.com/99768011/156433341-1202bf0f-1d77-49fb-b8ef-6ac05be95713.png">
> 
> Okay let's try to use the default InfuraProvider for now then (no parameters), it seems like we need to fix our Infura provider infrastructure first

updated the `ensProvider` to accept no parameters

I'll wait for an update from you then on the infura provider 🙂